src/framecontainer/rendering-modules/ViewerRenderProxy.ts

```diff
@@ -0,0 +1,55 @@
+import { FrameContainer } from '../FrameContainer';
+import { Messaging } from '@ampproject/viewer-messaging';
+import { TemplateProxyRequest, TemplateProxyResponse } from './ProxyRequest';
+import { Config } from '../../config';
+import { isValidURL, postJSON } from '../../util';
```

Comment:
where are these defined? I can't find them in this PR or currently existing. Maybe you forgot to add them?

Reply:
Sorry for the confusion: this PR is on top of #36 (just merged) that I split from it.